.github/workflows/sync_branches_reusable_workflow.yml

```diff
@@ -4,11 +4,17 @@ on:
   workflow_call:
     inputs:
       source-branch:
-        required: true
+        required: false
         type: string
+        default: main
       target-branch:
-        required: true
+        required: false
+        default: ananya-do-not-use-tmp
```

Comment:
Wouldn't `tmp-sync-test` be a better default name than something starting with `ananya-...` ?